src/coreclr/src/vm/jitinterface.cpp

```diff
@@ -10091,7 +10114,7 @@ void CEEInfo::getEEInfo(CORINFO_EE_INFO *pEEInfoOut)
     // Wrapper delegate offsets
     pEEInfoOut->offsetOfWrapperDelegateIndirectCell = OFFSETOF__DelegateObject__methodPtrAux;
 
-    pEEInfoOut->sizeOfReversePInvokeFrame  = (DWORD)-1;
+    pEEInfoOut->sizeOfReversePInvokeFrame = sizeof(FastReversePInvokeFrame);
```

Comment:
For crossgen, we should make this a constant `READYTORUN_ReversePInvokeTransitionFrameSizeInPointerUnits`, and follow the same pattern for it as for `READYTORUN_PInvokeTransitionFrameSizeInPointerUnits`.

This constant should include some extra space to give us version resiliency flexibility, same as `READYTORUN_PInvokeTransitionFrameSizeInPointerUnits`. I think making `2` would be reasonable.